metrics/cgroups/cgroups.go

```diff
@@ -70,6 +70,8 @@ func (m *cgroupsMonitor) Monitor(c runtime.Task) error {
 
 func (m *cgroupsMonitor) Stop(c runtime.Task) error {
 	info := c.Info()
+	t := c.(*linux.Task)
+	m.collector.LastCollect(info.ID, info.Namespace, t.Cgroup())
```

Comment:
How accurate are the metrics at this point when the task has already stopped running and you more than likely have an empty cgroup?

Reply:
From my testing, I've got accurate cumulative metrics (pages in/out/faults, cpu total/user/kernel time); I guess it depends on whether runc had the time to clean the cgroups or not.